src/Microsoft.Azure.WebJobs.Host/Loggers/UpdateParameterLogCommand.cs

```diff
@@ -18,11 +19,12 @@ internal sealed class UpdateParameterLogCommand : IRecurrentCommand
         private readonly IReadOnlyDictionary<string, IWatcher> _watches;
         private readonly IStorageBlockBlob _parameterLogBlob;
         private readonly TraceWriter _trace;
+        private readonly ILogger _logger;
 
         private string _lastContent;
 
         public UpdateParameterLogCommand(IReadOnlyDictionary<string, IWatcher> watches,
-            IStorageBlockBlob parameterLogBlob, TraceWriter trace)
+            IStorageBlockBlob parameterLogBlob, TraceWriter trace, ILogger logger)
```

Comment:
I think it will be a long time before we end up removing TraceWriter, so we'll be living with this for a while. It would be good if we can find a way to remove all this duplication until then,